target/linux/ath79/dts/qca9558_sitecom_wlr-8100.dts

```diff
@@ -0,0 +1,193 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca9557.dtsi"
+
+/ {
+	model = "Sitecom WLR-8100 v1 002 (X8 AC1750)";
```

Comment:
Can you give some details on the revision "systematics":
Will "v1 001" be flashable with the same image? Do we have to expect a v2?
Depending on the answer, we will have to call this device
sitecom,wlr-8100-v1
or
sitecom,wlr-8100-v1-002

Reply:
There are 2 version: WLR-8100 v1 001 and WLR-8100 v1 002.
I own the 002, so I don't know if it's compatible with 001 or not, but both versions seems to share the same firmware:
![image](https://user-images.githubusercontent.com/2470733/68243605-1da5b880-0013-11ea-9744-a52ca55f18fb.png)
I don't think we will see a newer revision of this router... as far as I know Sitecom doesn't sell anymore routers but never say never.
However I renamed it wlr-8100-v1-002.

Reply:
Well, if the firmware is exactly identical, I'd say we do not have to account for the difference.
So, it doesn't make sense to include -001 or -002 IMO.
If you are fairly sure that we won't see a v2, I'd be fine even without the -v1 as we have already introduced the wlr-7100 without one. In this case, please also remove the "v1 002" from model and DEVICE_MODEL.

Reply:
Ok. Now it should be ok